automation/devops_automation_infra/installers/k3s.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import time
+# import k8s
```

Comment:
CTRL + ALT + O

Reply:
BEWARE!
firstofall I dontthink it will remove commented imports, but as well pycharm thinks all Plugin imports are unused and will therefore remove them, so not recommended specifically here :)